api/helpers/uploadedFileErrorChecks.js

```diff
@@ -30,13 +30,17 @@ async function connectToClamAV(req) {
 }
 
 function initialiseClamScan(req) {
-    const { clamav_host: clamavHost, clamav_port: clamavPort } =
-        req._sails.config.upload;
-
+    const {
+        clamav_host: clamavHost,
+        clamav_port: clamavPort,
+        clamav_debug_enabled: clamavDebugEnabled,
+    } = req._sails.config.upload;
     const clamAvOptions = {
+        debugMode: JSON.parse(clamavDebugEnabled),
         clamdscan: {
             host: clamavHost,
             port: clamavPort,
+            active: true,
```

Comment:
What does active mean? There's also clamav_enabled, does it do the same thing?

Reply:
This works a bit differently from `clamav_enabled ` but it's actually pretty minor so I can remove it if necessary.

With **debug mode** turned on it becomes obvious that clamscan tries to search for both the clamscan and clamdscan binaries which don't exist because we are using ClamAV via a socket/TCP connection, not locally.

<img width="898" alt="Screenshot 2022-01-25 at 16 44 16" src="https://user-images.githubusercontent.com/1377253/151023068-39669db2-4ab6-4dd8-be1d-0fe25bff88ca.png">

I thought setting active: true would fix this but it should actually be `false` (https://github.com/kylefarris/clamscan#getting-started). 🤦

I'll push that fix soon.